spec/fixtures/lab/outside.js

```diff
@@ -0,0 +1,5 @@
+describe('Addition', function() {
+  it('adds two numbers', function() {
+    assertEqual(1 + 1 == 2)
+  });
+});
```

Comment:
We don't need this file for tests, Vim can `view outside.js` even if the file doesn't exist – in that case it opens a new buffer.